CHANGELOG.md

```diff
@@ -1,13 +1,17 @@
-# websitetoolboxforum Changelog
+# websitetoolboxcommunity Changelog
 
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## 1.4.0 - 2023-02-14
+### Added
+- Make Website Toolbox Community plugin work with craftcms version4. 
```

Comment:
Missing space

Reply:
Done

Reply:
Obviously the change is for the WT community plugin. What else would it be for? This is not a good comment. Change to "Added support for CraftCMS 4".

Make sure user-facing things look professional.